### DISCUSSION TITLE
Allow to set auto logout after some time from user interface

### DISCUSSION THREAD

**Discussion by gbertolottiKS** [nextcloud/all-in-one#1489] [Questions] [answered]
Hi.
We wanted to set an option that do an automatic logout after 8 hours of inactivity of a certain account.
We searched online, and we found this documentation: https://docs.nextcloud.com/server/latest/admin_manual/configuration_server/config_sample_php_parameters.html#user-experience
So, we changed the config.php file, appending the lines:
```
'session_lifetime' => 60 * 60 * 8,
'auto_logout' => true,
```
but we had to roll back, because it didn't work as expected (some users claims that Nextcloud kicks them out after only few minutes, for others nothing changed).
We have 2 doubt:

1. was the correct configuration (to achieve what we want)? If so, there is a bug somewhere..
2. shouldn't exist a more user friendly option from the admin option UI?

Thanks

**Comment by szaimen** [comment, marked as answer]
Hi, please ask your question here: https://help.nextcloud.com
Thank you!
